Automattic/jetpack
# PHPCS: SAL (part 3)

This PR partially fixes the issue at #21967

#### Changes proposed in this Pull Request:

* This PR updates the SAL files to meet PHPCS standards:
  * "projects/plugins/jetpack/sal/class.json-api-site-jetpack-base.php" 
  * "projects/plugins/jetpack/sal/class.json-api-site-jetpack.php" 
  * "projects/plugins/jetpack/sal/class.json-api-token.php" 

Adding here as well that I'll approach the WordPress.com files separately.

#### Jetpack product discussion

n/a.

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* For each file, test features that may be impacted by that file.
* Also, you can use the WordPress.com developer console to test an endpoint such as the WP.COM API GET request for `/sites/{blog_id}/posts/{post_id}` - for a sandboxed WPcom site and a sandboxed Jetpack site. A breaking change will affect the output, whereas it should work fine otherwise.
* We're primarily checking for any broken experiences or PHPCS errors.
